examples/bin/cli.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+java -Xmx256m -Duser.timezone=UTC -Dfile.encoding=UTF-8 \
+        -cp lib/*:config/_common:config/broker \
```

Comment:
Would you tell me why it includes the broker configuration?

Reply:
corrected